test_sqlalchemy.py

```diff
@@ -503,6 +503,16 @@ class FooBoundModel(AbstractFooBoundModel):
         self.assertEqual(len(metadata.tables), 1)
         self.assertTrue('foo_bound_model' in metadata.tables)
 
+    def test_extended_engine_options(self):
+        app = flask.Flask(__name__)
+        app.config['SQLALCHEMY_BINDS'] = {
+            'foo': 'sqlite://'
+        }
+        db = fsa.SQLAlchemy(app, engine_options=dict(echo=True))
```

Comment:
Probably not the best example for a unit test considering this one can be set through the flask config (`SQLALCHEMY_ECHO`)